pkg/process/config/config.go

```diff
@@ -441,6 +433,24 @@ func loadEnvVariables() {
 	}
 }
 
+func loadSysProbeEnvVariables() {
+	for _, variable := range []struct{ env, cfg string }{
+		{"DD_SYSTEM_PROBE_ENABLED", "system_probe_config.enabled"},
+		{"DD_SYSPROBE_SOCKET", "system_probe_config.sysprobe_socket"},
+		{"DD_SYSTEM_PROBE_CONNTRACK_IGNORE_ENOBUFS", "system_probe_config.conntrack_ignore_enobufs"},
+		{"DD_DISABLE_TCP_TRACING", "system_probe_config.disable_tcp"},
+		{"DD_DISABLE_UDP_TRACING", "system_probe_config.disable_udp"},
+		{"DD_DISABLE_IPV6_TRACING", "system_probe_config.disable_ipv6"},
+		{"DD_DISABLE_DNS_INSPECTION", "system_probe_config.disable_dns_inspection"},
+		{"DD_COLLECT_LOCAL_DNS", "system_probe_config.collect_local_dns"},
```

Comment:
Isn't this handle by default by viper ?

Reply:
For future reference: 
- This is not handled by viper because the expected names that get passed in do not work with how we set up viper to read env vars. For example,  "DD_SYSTEM_PROBE_ENABLED" would need to be "DD_SYSTEM_PROBE_CONFIG_ENABLED" - notice the inclusion of "CONFIG'.